modules/op-tkn-hub-interaction.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// *  cli_reference/tkn_cli/op-tkn-reference.adoc
+
+[id="op-tkn-hub-interaction_{context}"]
+= Hub interaction commands
+
+Interact with Tekton Hub for resources such as tasks and pipelines.
+
+== hub
+Interact with hub.
+
+.Example: Display help
+[source,terminal]
+----
+$ tkn hub -h
+----
+
+.Example: Interact with hub API server
+[source,terminal]
+----
+$ tkn hub --api-server https://api.hub.tekton.dev
+----
+
+[NOTE]
+====
+For each command examples given below, you can get the corresponding sub-commands and flags by executing `tkn hub <command> --help`.
+====
+
+== hub downgrade
+Downgrade an installed resource
+
+.Example: Downgrade an installed resource
+[source,terminal]
+----
+$ tkn hub downgrade task -n mynamespace --to <resourceVersion>
+----
+
+== hub get
+Get resource manifest by its name, kind, catalog, and version.
+
+.Example: Get resource manifest
```

Comment:
Get resource manifest of task `mytask` of version `0.2`  from `tekton` catalog

Reply:
same here.. instead of giving number to version we can use `newversion`
WDYT?